products/anolis8/transforms/xccdf2table-profileanssirefs.xslt

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
Dtto. The `xccdf2table-profileanssirefs.xslt` doesn't exist elsewhere and it seems to not be used by this PR.